bin/compile

```diff
@@ -171,7 +171,13 @@ BIN_DIR=$BUILD_DIR/.apt/usr/bin
 rm $BIN_DIR/$SHIM
 cat <<EOF >$BIN_DIR/$SHIM
 #!/usr/bin/env bash
-exec \$HOME/.apt/opt/google/$BIN --headless --no-sandbox --disable-gpu --remote-debugging-port=9222 \$@
+if [ $1 = "--version" ]; then
```

Comment:
Shouldn't the $ in `$1` be escaped ?
When using the heroku/google-chrome buildpack and running `heroku ci:debug`, I get
```
~ $ cat $GOOGLE_CHROME_SHIM   
#!/usr/bin/env bash
if [ /app = "--version" ]; then
  exec $HOME/.apt/opt/google/chrome/chrome --version
elif [ /app = "--product-version" ]; then
  exec $HOME/.apt/opt/google/chrome/chrome --product-version
else
  exec $HOME/.apt/opt/google/chrome/chrome --headless --no-sandbox --disable-gpu --remote-debugging-port=9222 $@
fi
~ $ 
```

Reply:
Yes this is correct, I'm seeing this:
```
if [ /tmp/build_9b8b699572a1f61d9096ed0e275f2c41 = "--version" ]; then
```
because the `$1` isn't escaped.

Reply:
Fixed in #74